config.go

```diff
@@ -345,7 +347,7 @@ func loadAndValidateConfig(interceptor signal.Interceptor) (*Config, error) {
 	// Show the version and exit if the version flag was specified.
 	appName := filepath.Base(os.Args[0])
 	appName = strings.TrimSuffix(appName, filepath.Ext(appName))
-	if preCfg.Lnd.ShowVersion {
+	if preCfg.Lnd.ShowVersion || preCfg.ShowVersion {
```

Comment:
seems the version number to be reported should be exclusively related to litd, when passing the `--version` flag, rather than lnd.

E.g.
Instead of lnd's 0.18.2 being reported here:
```shell
$ ./litd-debug --version
litd-debug version 0.18.2-beta commit=lightning-terminal-v0.13.3-alpha-1-gc416ca21a056d35506976315ba7d65c5f26dee2c
```

It should be
```shell
$ ./litd-debug --version
litd-debug version 0.13.4-alpha
```

e.g. `0.18.2-beta` changed to `0.13.4-alpha`

Reply:
We've always had the combined version of lnd and litd.